### ISSUE TITLE
Have mainline builds for Inkscape include these symbols sets

### ISSUE THREAD

**Issue by PanderMusubi** [Xaviju/inkscape-open-symbols#19]
Please discuss with Inkscape maintainers to include these symbols sets in main builds for all platforms. see also https://bugs.launchpad.net/inkscape/+bug/1464270

**Comment by Xaviju** [comment]
Thats great! I would love to see this icons, or some of them, as default inkscape symbols!
I think I will add a message in the README so users can vote if they want this icons to be in future inkscape versions.

**Comment by victorwestmann** [comment]
I agree. Having these good looking symbols embedded in Inkscape would be a great addition to the project. :+1:

**Comment by Xaviju** [comment]
No news about this. We might need to make some noise online? 
There are a few issues regarding the symbol menu that could be adressed as well. I wish Inkscape could work on this since I have no idea of how to do it myself :(

**Comment by Moini** [comment]
If someone wants to ask the devs, there's the development mailing list. I guess it wouldn't be very difficult to actually add the symbols to Inkscape, code-wise.

However, this would create lots and lots of work for translators, and also slows down the symbols dialog loading quite a bit.
So maybe having a separate package that can be installed somehow would make more sense?

sozi and ink-generator packages (available for Ubuntu) copy their files into the /usr/share/inkscape/extensions directory (probably not a very clean way to do it, but the only feasible option with Inkscape, which doesn't support adding custom source directories).

**Comment by Xaviju** [comment]
@Moini I understand. I saw that someone created something similar in AUR https://aur.archlinux.org/packages/inkscape-open-symbols-git/ but I have no idea how to create that in Ubuntu. That would be helpful.
Regarding the symbols in Inkscape, I would suggest changing the default inkscape icons for a more applicable Open Source icon pack. I've never used them. But this is my opinion, do you use the default isymbol set in Inkscape?

**Comment by Moini** [comment]
Tbh. I've never really used any of the symbols... neither 'ours', nor Inkscape's stock symbols (where is that 'embarrassed' emoji when you need it?...). I guess they are most useful when you're either making a map, or planning a web page. I've done both, but never at a scale that required symbols.

Making a .deb package isn't really so hard. The hard part is asking for inclusion in official repos, as far as I know - and finding someone who would be your sponsor. I'm not sure how it works for Debian/Ubuntu.

Making a ppa is within our reach, though, without having to rely on external support. That would take a launchpad account + team, and then someone who is courageous enough to wade through their settings.
I've never done that, and also never built a real package that isn't its own program, but just files that need to be copied into the correct directories, but I know someone who can possibly help at least with the lp setup, so I might dare do that (if desired).

I'm not sure I understand the part about the icons. Do you mean Inkscape's icons.svg? The icons for Inkscape's own buttons? Or were you talking about replacing the stock symbols, @Xaviju ?

**Comment by Xaviju** [comment]
Thanks @Moini 
About the PPA, thanks for explaining the possibilities. Then, I think is not worth the effort. For non-technical audience is easier to copy some files into a directory than to use an external PPA. I would rather spend our energy in something that make installing the icons plain simple (click, click) if ever exist.
Yes, I was suggesting replacing the current Inkscape stock symbols aiming for the same goal: offering nice symbol sets by default. I do use some of _our_ sets and I think are diverse enough to satisfy many users. My feeling is that having 3-4 of _our_ sets would be more useful, but I'm not sure either, just wanted to listen your opinions before bothering inkscape devs.

**Comment by PanderMusubi** [comment]
`apt-get install inkscape-open-symbols` is my preferred weapon of choice. Just make sure to submit a new version before each Ubuntu release freeze.

**Comment by Moini** [comment]
@Xaviju : Ah, okay. Yes, I can't comment much on that - but my guess is that once something is /in/ Inkscape, someone will miss it when it's gone. I don't know how devs think about it.

Once a ppa has been added, updating stuff would be simple (possibly even automatical). Adding a ppa requires you to copy-paste a line into your settings, via UI (on Ubuntu and Linux Mint). For Ubuntu/Ubuntu-based systems, we could probably also just write a shell script that users need to run (but I'm not sure that's actually quicker than copying things into /home/username/.config/inkscape/symbols).

@PanderMusubi : Yes, that would be nice. Do you have any contacts to the package maintainer crowd?

**Comment by Xaviju** [comment]
@Moini thanks! I didn't know that there was a UI for adding PPAs!! So useful! Then yes I would go with the PPA :)
@PanderMusubi How could we do that? Could some users install it via Ubuntu Software Center?

**Comment by PanderMusubi** [comment]
@Xaviju At the moment it is in the Debian and Ubuntu repositories. Please, contact Félix Sipma felix+debian@gueux.org and CC ubuntu-devel-discuss@lists.ubuntu.com and pkg-multimedia-maintainers@lists.alioth.debian.org

**Comment by Moini** [comment]
@PanderMusubi : I found it, it's in zesty and artful repos! When has it been added? 
(Not available for me, unfortunately, on xenial.)

**Comment by PanderMusubi** [comment]
https://launchpad.net/ubuntu/+source/inkscape-open-symbols

**Comment by Xaviju** [comment]
Sorry @PanderMusubi and @Moini I'm not an Ubuntu or Debian user and I'm not sure if I understood well. Do this means that inkscape open symbols will be available to install via `apt-get` or in the Software center in the Ubuntu 17.04 (Zesty Zapus) release?

**Comment by Moini** [comment]
Yes, it's available for Zesty, and will be available for Artful (version for October). Both are not Long Time Support releases, so probably fewer people will use them.

**Comment by Xaviju** [comment]
That is so awesome!!! Thanks guys! This is so cool for open source designers!!

**Comment by Moini** [comment]
(I didn't do anything - it already happened in January. No idea whose initiative it was, in the end. And not sure what to do with updates, and how often...)

**Comment by Xaviju** [comment]
I have no idea what should we do either. Anyway, thanks!
I'll contact Félix Sipma as @PanderMusubi suggested and let you know in this issue.

**Comment by PanderMusubi** [comment]
I'd suggest to release a version before end of each January and end of each July in order to make it in the next Ubuntu release. Backports and updates are also possible to provide throughout the year. Please fine-tune this with the current Ubuntu package maintainer.

**Comment by PanderMusubi** [comment]
I'd suggest to release a version before end of each January and end of each July in order to make it in the next Ubuntu release. Backports and updates are also possible to provide throughout the year. Please fine-tune this with the current Ubuntu package maintainer.

**Comment by PanderMusubi** [comment]
Here is the schedule for the next release https://wiki.ubuntu.com/BionicBeaver/ReleaseSchedule Please, see of you can do another release in January.

**Comment by Xaviju** [comment]
Many improvements in the recent weeks. Makes sense to release again. I'll try to add a few improvements before the new release. Thanks for the heads up.

**Comment by PanderMusubi** [comment]
Christmas release for under the tree? ;-)

By the way, I don't know how requesting backports work. That might be good to ask as well to the person which upgrades the Debian/Ubuntu package. Then the new release will reach much more users.

**Comment by Xaviju** [comment]
New release with lots of fixes under the hood and many updated symbols! 
https://github.com/Xaviju/inkscape-open-symbols/releases/tag/v1.2

**Comment by PanderMusubi** [comment]
Has to be discussed with Inkscape via https://gitlab.com/inkscape/inkscape/-/issues
